### PR TITLE
chore: remove `core.metrics()`

### PR DESCRIPTION
**Pull request by iuioiua**
As `Deno.metrics()` is now removed from the runtime. Disclaimer: I'm not very familiar with this codebase. I'm not sure if there are other things that could be cleaned up.

### CODE REVIEW

File: core/ops_metrics.rs [hunk 1]
```diff
@@ -116,76 +113,3 @@ impl OpMetricsSummary {
     self.ops_dispatched_async > self.ops_completed_async
   }
 }
-
-#[derive(Default, Debug)]
-pub struct OpMetricsSummaryTracker {
```
Comment (iuioiua) [inline core/ops_metrics.rs:121]: IIUC, `core.metrics()` was the only symbol that used this struct.

**Comment by irbull** [comment]
I've filed a PR for the failing builds in deno_core. https://github.com/denoland/deno_core/pull/895

File: core/ops_metrics.rs (hunk 1 above)
Comment (bartlomieju) [reply core/ops_metrics.rs:121]: I believe this is actually used by the `deno test` for leaking ops discovery. This should most likely stay. See `runtime/worker.rs` in Deno repo.

**Review by bartlomieju — CHANGES_REQUESTED** [review]

**Review by bartlomieju — APPROVED** [review]

**Review by bartlomieju — APPROVED** [review]

**Comment by iuioiua** [comment]
Please merge, @bartlomieju

**Pull request merged**